### ISSUE TITLE
Alignment started with a customized GTF on some, but not all FASTQs

### ISSUE THREAD

**Issue by AlfredoLebron** [abl0719/ASElux#10]
Hello. I'm trying to detect ASE using a GTF I constructed from regions of expression from RNA sequencing data (PROseq). ASElux starts alignment on two donor FASTQs with an index from my GTF and successfully gives an output. On the other donor FASTQ, however, a segmentation fault error is given several minutes after building a dynamic index. The FASTQ donor that does not work with my custom GTF still works with a standard hg38 annotation, so that leads me to believe there is an issue with my GTF. Here is the head of my GTF:
```
chr1	ASElux	gene	100783049	101419549	.	+	.	.
chr1	ASElux	transcript	100783049	101419549	.	+	.	.
chr1	ASElux	exon	100783049	101419549	.	+	.	.
chr1	ASElux	gene	101416349	101507649	.	-	.	.
chr1	ASElux	transcript	101416349	101507649	.	-	.	.
chr1	ASElux	exon	101416349	101507649	.	-	.	.
chr1	ASElux	gene	101430249	101431149	.	+	.	.
chr1	ASElux	transcript	101430249	101431149	.	+	.	.
chr1	ASElux	exon	101430249	101431149	.	+	.	.
chr1	ASElux	gene	101442349	101642399	.	+	.	.
```
I generated the start/end coordinates based on predicted transcription units and I had to manually set up the GTF file from those coordinates. The index could only be built if I included "gene", "transcript", and "exon", hence the repeating rows.

Is there any reason why an index built with my GTF could work with some but not all FASTQs? Thank you.

**Comment by AlfredoLebron** [comment]
I would like to leave an update. I found out that the issue I'm having is not specific to the donors. After segmenting my GTF by chromosome, I found out that ASElux threw a "segmentation fault" error when given chromosomes 14 and 15 as indices. I'm currently trying to figure out why this is the case. The coordinates of my custom GTF are within the standard hg38 chromosome size coordinates. Is it possible that ASElux won't work with a GTF with a very large coverage of a chromosome, and do the coordinates need to be in a specific range in a chromosome? Thank you.
